### PR TITLE
metrics: display range of values

### PR DESCRIPTION
**Pull request by terceiro**

### CODE REVIEW

**Comment by terceiro** [comment]
Screenshot_2019-06-27 SQUAD -

File: squad/frontend/static/squad/controllers/charts.js [hunk 1]
```diff
@@ -231,16 +229,50 @@ function ChartPanel($http, DATA) {
...
+                        label: ':low:' + env.name,
+                        data: low,
+                        backgroundColor: env.range_color,
+                        pointRadius: 0,
+                        pointHoverRadius: 0,
+                        borderColor: '#ffffff00', // transparent
+                        yidx: 6,
+                        fill: '+1'
+                    }
+                    )
```
Comment (chaws) [inline squad/frontend/static/squad/controllers/charts.js]: i think this parenthesis should've been in line above, right after closing curly braces

File: squad/frontend/static/squad/controllers/charts.js [hunk 2]
```diff
@@ -683,6 +725,19 @@ function ChartsController($scope, $http, $location, $compile, ChartPanel, DATA)
         })
     }
 
+    $scope.filterByDate = function(chartPanel, minLimit, maxLimit) {
+        console.log(chartPanel)
```
Comment (chaws) [inline squad/frontend/static/squad/controllers/charts.js]: leftover debugging...

**Comment by chaws** [comment]
I noticed that there's an if condition to check whether `low != line` and `high != line`, but that only worked for integer values.

In the screenshot below, it shows ':low:env' and ':high:env' displaying same values

Screenshot from 2019-07-01 17-26-37

File: squad/frontend/static/squad/controllers/charts.js (hunk 1 above)
Comment (terceiro) [reply squad/frontend/static/squad/controllers/charts.js]: fixed

File: squad/frontend/static/squad/controllers/charts.js (hunk 2 above)
Comment (terceiro) [reply squad/frontend/static/squad/controllers/charts.js]: yes, nice catch

**Comment by terceiro** [comment]
> I noticed that there's an if condition to check whether `low != line` and `high != line`, but that only worked for integer values.

actually it doesn't work for arrays at all:

**Pull request merged**